src/main/resources/db/migration/all/20240221091158__add_created_from_appeal_to_assessments.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE approved_premises_assessments ADD COLUMN created_from_appeal BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
shouldn't this be DEFAULT FALSE, otherwise all existing assessments will be 'created from appeal'?

Reply:
Good point!